src/functionalTest/resources/features/F-705/F-705.feature

```diff
@@ -0,0 +1,132 @@
+#================================================
+@F-705
+Feature: Verify microservice based authorisation
+#================================================
+
+
+Background:
+  Given an appropriate test context as detailed in the test data source
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# CPO-10 / AC-1
+@S-705.1
+Scenario: Must successfully allow access of the AddCasePaymentOrder API for a CRUD whitelisted service
```

Comment:
Should we rename all the references to AddCasePaymentOrder to the CreateCasePaymentOrder that we use elsewhere?

Reply:
Will take a look

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/68/commits/970b0c16e75dc6be385b97fa1c0dd42e08d20d2b